WalletWasabi.Fluent/Views/AddWallet/ConnectHardwareWalletView.axaml

```diff
@@ -0,0 +1,68 @@
+
+    <UserControl.Styles>
+        <Style Selector="StackPanel.Delay">
+            <Setter Property="Opacity" Value="0"/>
+            <Style.Animations>
+                <Animation Duration="0:0:30">
+                    <KeyFrame Cue="0%">
+                        <Setter Property="IsVisible" Value="False"/>
+                    </KeyFrame>
+                    <KeyFrame Cue="100%">
+                        <Setter Property="IsVisible" Value="True"/>
+                    </KeyFrame>
+                </Animation>
+            </Style.Animations>
+            <Setter Property="Transitions">
+                <Transitions>
+                    <DoubleTransition Property="Opacity" Duration="0:0:0.5" />
+                </Transitions>
+            </Setter>
+        </Style>
+        <Style Selector="StackPanel.Delay[IsVisible=True]">
+            <Setter Property="Opacity" Value="1"/>
+        </Style>
+    </UserControl.Styles>
+
+    <c:ContentArea Title="Hardware wallet"
+                   Caption="The hardware wallet will automatically appear when connected to the computer. The detection may take some time."
+                   EnableCancel="True" CancelContent="Cancel"
+                   EnableBack="True"
+                   EnableNext="True" NextContent="Connect" >
+      <DockPanel>
+
+          <StackPanel Spacing="8" DockPanel.Dock="Bottom" Margin="0 15 0 0">
+              <StackPanel Classes="Delay">
+                  <TextBlock Text="Device did not show up?" FontWeight="Bold" />
+                  <TextBlock Text="- Be sure to set up and log into the device, usually with a PIN." TextWrapping="Wrap" />
+                  <TextBlock Text="- Hardware wallets are picky about their USB cables, so try out multiple ones." TextWrapping="Wrap" />
+                  <StackPanel IsVisible="{h:Platform {x:Static h:Const.False}, Linux={x:Static h:Const.True}}">
+                      <TextBlock Text="- Make sure that you properly applied udev rules. These are necessary for the devices to be reachable on Linux environments:" TextWrapping="Wrap" />
+                      <Button Content="https://github.com/bitcoin-core/HWI/tree/master/hwilib/udev" Classes="activeHyperLink" Command="{Binding OpenBrowserCommand}" CommandParameter="{Binding $self.Content}" />
+                  </StackPanel>
+              </StackPanel>
+             <StackPanel>
+                  <TextBlock Text="Limitations?" FontWeight="Bold" />
+                  <TextBlock Text="- Currently all hardware wallets are incompatible with coinjoins." />
+                  <TextBlock Text="- Only bech32 keypaths are supported, your legacy hardware wallet transactions won't show up." />
+              </StackPanel>
+              <StackPanel>
+                  <TextBlock Text="Which hardware wallets are supported?" FontWeight="Bold" />
+                  <TextBlock Text="- Coldcard, Ledger Nano S and Trezor T." TextWrapping="Wrap" />
```

Comment:
Trezor T is not officially supported https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md#officially-supported-hardware-wallets

Reply:
@nopara73 asked me IRL to put there Trezor T too.

Reply:
@soosr Thanks, just wanted to make sure that you are aware of that

Reply:
Next time I will leave a comment or something 😀